x-pack/plugins/infra/public/pages/logs/stream/page_toolbar.tsx

```diff
@@ -121,15 +120,21 @@ export const LogsToolbar = () => {
           />
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <LogTimeControls
-            currentTime={visibleMidpointTime}
-            isLiveStreaming={isAutoReloading}
-            jumpToTime={jumpToTargetPositionTime}
-            startLiveStreaming={() => {
-              startLiveStreaming();
-              setSurroundingLogsId(null);
+          <EuiSuperDatePicker
+            start={startDate}
+            end={endDate}
+            onTimeChange={handleTimeChange}
+            isPaused={!isStreaming}
+            refreshInterval={liveStreamingInterval}
+            onRefresh={() => {}}
```

Comment:
Might this be the reason why clicking "Refresh" doesn't refresh the data? The log rate and log category tabs are examples of how this could be achieved.